### ISSUE TITLE
Project with dedicated `tests/` vs. `src/` directories errors with prefix exception

### ISSUE THREAD

**Issue by filmaj** [denoland/dnt#249]
*TL;DR* [A CI execution for our deno project](https://github.com/slackapi/deno-slack-sdk/actions/runs/4056039718/jobs/6979995034) which uses `dnt` to compile a version of the project into a node-friendly format failed after I introduced a new `tests/` directory at the top level of our project, while the entrypoint for the project is under `src/mod.ts`. The error is:

```
[dnt] Transforming...
error: Uncaught (in promise) "Error stripping prefix of /Users/fmaj/src/deno-slack-sdk/tests/integration/functions/function_handler_test.ts with base /Users/fmaj/src/deno-slack-sdk/src"
```

## Many Thanks

Before diving in, many thanks for the excellent project! Much appreciated ❤️ 

## Longer Version

Our project's entry point has always been `src/mod.ts`, and tests were initially sprinkled around under `src/` in various places and used the deno convention of `_test.ts` file name suffixes to differentiate between source vs. test files.

Recently we worked on a refactor of the project and we decided to keep more unit-test-y files under `src/` as before, while we introduced more integration-test-y files in a new directory in the root of the project under `tests/` - outside of the `src/` directory.

The full code / build options for our `dnt` use can be found [here](https://github.com/slackapi/deno-slack-sdk/blob/main/build_npm.ts).

I can work around this issue by setting `test: false` on the dnt build options, FYI, but it seems like housing tests outside of any directory referenced in the entry point is not supported by dnt?

**Comment by bdarcus** [comment]
I ran into the same issue.

**Comment by eythaann** [comment]
2 years later and this issue still is not resolved
